workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -84,6 +85,11 @@ internal static string GetThisPackagePath()
             }
         }
 
+        private static string[] GetCommonMacPaths()
```

Comment:
This may be better initialized as a static readonly array, so we don't have to re-allocate it every time.